src/main/java/appeng/core/api/definitions/ApiBlocks.java

```diff
@@ -557,7 +568,7 @@ private static IBlockDefinition makeSlab( String slabId, String doubleSlabId, Fe
 		Block block = blockDef.maybeBlock().get();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Call "Optional#isPresent()" before accessing the value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS3655)